src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -40,6 +40,7 @@ public String getConnectionURI() {
 
 	private OptionParser getOptionParser() {
 		OptionParser parser = new OptionParser();
+		parser.accepts( "log_level", "log level, one of DEBUG|INFO|WARN|ERROR" ).withRequiredArg();
```

Comment:
it is optional, yeah.  the default is whatever gets set in the xml file.  saying `withRequiredArg` means that `--log_level` takes exactly 1 arguments, not that `--log_level` is a required argument...